src/foam/nanos/alarming/AlarmLoggingDAO.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.alarming',
+  name: 'AlarmLoggingDAO',
+  extends: 'foam.dao.ProxyDAO',
+
+  documentation: `Generate Logger messages for each alarm`,
+
+  javaImports: [
+    'foam.nanos.logger.Logger',
+  ],
+
+  methods: [
+    {
+      name: 'put_',
+      javaCode: `
+      Alarm alarm = (Alarm) getDelegate().put_(x, obj);
+      Logger logger = (Logger) x.get("logger");
+      switch ( alarm.getSeverity().getName() ) {
+        case "DEBUG":
```

Comment:
Can the switch case be on the enum instead of the name?
```suggestion
      switch ( alarm.getSeverity() ) {
        case DEBUG:
```

Reply:
yes